Exercise 6 text: suggested improvements

aral opened the pull request.

aral committed:
Punctuation and grammar improvements: first paragraph

aral committed:
s/users/people/ – makes it sound more humane; prefer when possible

aral committed:
Minor improvement to sentence structure; punctuation

aral committed:
Make link to dns-discovery module consistent with rest; uppercase DNS

aral committed:
Improve formatting, grammar in the connection strategies section

aral committed:
Reword and attempt to clarify the bootstrap nodes section

aral committed:
Format file name in italics instead of in code style

aral committed:
Update formatting of file name; attempt to improve clarity

okdistribute commented on problems/06.md:
```diff
@@ -1,26 +1,25 @@
 ### 6: Networking with other peers
 
-Great, we now how a working chat app however it is not that interesting yet. First of all we can only chat with ourself and even worse we can only see the data in the same process we are writing. Kinda boring.
+Great, we now how a working chat app! However, it is not that interesting yet. First of all, we can only chat with ourselves. Even worse, we can only see the data in the same process that we are writing to. Kinda boring.
```
to which we are writing

okdistribute commented on problems/06.md:
```diff
@@ -1,26 +1,25 @@
-3. If these methods fail, it looks in a DHT (_distributed hashtable_). For this, `discovery-channel` uses [bittorrent-dht](https://github.com/webtorrent/bittorrent-dht).
+1. Search the local network. `discovery-channel` uses a module called [`dns-discovery`](https://github.com/mafintosh/dns-discovery) that has the ability to make queries in the local network using multicast DNS (MDNS).
+2. Search the internet using a well-known server. The next phase is to search using DNS, also with `dns-discovery`.
+3. If these methods fail, look in a DHT (_distributed hashtable_). For this, `discovery-channel` uses [`bittorrent-dht`](https://github.com/webtorrent/bittorrent-dht).
```
it looks

okdistribute commented on problems/06.md:
```diff
@@ -1,26 +1,25 @@
-2. Search the internet using a well-known server. The next phase is to search using dns, also with `dns-discovery`.
-3. If these methods fail, it looks in a DHT (_distributed hashtable_). For this, `discovery-channel` uses [bittorrent-dht](https://github.com/webtorrent/bittorrent-dht).
+1. Search the local network. `discovery-channel` uses a module called [`dns-discovery`](https://github.com/mafintosh/dns-discovery) that has the ability to make queries in the local network using multicast DNS (MDNS).
+2. Search the internet using a well-known server. The next phase is to search using DNS, also with `dns-discovery`.
```
searches* probably

okdistribute commented on problems/06.md:
```diff
@@ -37,8 +36,8 @@ swarm.on('connection', function (connection, info) {
 
-* Try running discovery.js twice, each in its own terminal window. 
-* They should find each other if they're using the same swarm key.
+* Try running _discovery.js_ twice, each instance in its own terminal window.
```
just 'each'

okdistribute commented on problems/06.md:
```diff
@@ -37,8 +36,8 @@ swarm.on('connection', function (connection, info) {
-* Try running discovery.js twice, each in its own terminal window. 
-* They should find each other if they're using the same swarm key.
+* Try running _discovery.js_ twice, each instance in its own terminal window.
+* The two instances should find each other if they're using the same swarm key.
```
the two processes

aral committed:
Implement @karissa’s suggestion (“to which we are writing”)

Resolves https://github.com/kappa-db/workshop/pull/10#pullrequestreview-185371096

aral committed:
Implement the remainder of @karissa’s suggestions

  * Resolves https://github.com/kappa-db/workshop/pull/10#pullrequestreview-185371103
  * Resolves https://github.com/kappa-db/workshop/pull/10#pullrequestreview-185371107
  * Resolves https://github.com/kappa-db/workshop/pull/10#pullrequestreview-185371117
  * Resolves https://github.com/kappa-db/workshop/pull/10#pullrequestreview-185371119

okdistribute committed:
Merge branch 'master' into chapter-06-fixes
